packages/components/src/Card/src/Card.tsx

```diff
@@ -0,0 +1,81 @@
+import { useStyledSystem, type StyledComponentProps } from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { forwardRef, type CSSProperties, type ForwardedRef } from "react";
+import { HeaderContext, useContextProps } from "react-aria-components";
+
+import { ContentContext, FooterContext } from "../../layout/index.ts";
+import { cssModule, SlotProvider, type BaseComponentDOMProps } from "../../utils/index.ts";
+
+import { CardContext } from "./CardContext.ts";
+
+import styles from "./Card.module.css";
+
+export const GlobalCardCssSelector = "hop-Card";
+
+export interface CardProps extends StyledComponentProps<BaseComponentDOMProps> {
+    variant?: "main" | "second-level";
+}
+
+const Card = (props: CardProps, ref: ForwardedRef<HTMLElement>) => {
+    [props, ref] = useContextProps(props, ref, CardContext);
+    const { stylingProps, ...ownProps } = useStyledSystem(props);
+    const {
+        className,
+        children,
+        style,
+        slot,
+        variant = "main",
+        ...otherProps
+    } = ownProps;
+
+    const classNames = clsx(
+        GlobalCardCssSelector,
+        cssModule(
+            styles,
+            "hop-Card",
+            variant
+        ),
+        stylingProps.className,
+        className
+    );
+
+    const mergedStyles: CSSProperties = {
+        ...style,
+        ...stylingProps.style
+    };
+
+    return (
+        <section
```

Comment:
Do we foresee situations where someone wants something else than a section?

Reply:
I don't think we want to handle polymorphic components anymore, did you want to suggest something different?